hw/ip/otbn/dv/otbnsim/sim/insn.py

```diff
@@ -810,7 +810,7 @@ def __init__(self, op_vals: Dict[str, int]):
     def execute(self, model: OTBNModel) -> None:
         idx = self.wsr
         old_val = model.state.wcsr_read(idx)
-        new_val = old_val | model.state.wreg[self.wrs]
+        new_val = model.state.wreg[self.wrs] | old_val
```

Comment:
I don't think we actually need `new_val` to be a register. Can we just call `model.state.wreg[self.wrs].unsigned()` instead? It seems a lot less magic.